generative_ai/streaming_chat.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_streaming_chat]
+import vertexai
+from vertexai.language_models import ChatModel, InputOutputTextPair
```

Comment:
https://google.github.io/styleguide/pyguide#22-imports: `Use import statements for packages and modules only, not for individual classes or functions.`

Reply:
Unfortunately, if I don't import the line17, there will be error. It will say "NameError: name 'InputOutputTextPair' is not defined"